### PR TITLE
Content: Compenencies - Modeling

### PR DESCRIPTION
**Pull request by RGBD**
Current progress:
Added 
- "Overview"
- "Architectural Frameworks"
TODO:
- Add image examples for "Architecture Frameworks"
- Add "diagramming"
- Add "capturing change"
- Add "documenting"

Not final version, work in progress.

### CODE REVIEW

**Comment by RGBD** [comment]
New version uploaded. Notable changes:
- Added links and hilighting for catchphrases.
Chapters structure revised:
- Architecture framework examples nested udner "architectural frameworks"
- Chapters "Capturing Change, Documeting" dropped.

Removing draft state

File: content/docs/competencies/modeling/diagramming.md
Comment (ihardzeenka) [inline content/docs/competencies/modeling/diagramming.md:1]: Please consider shortening Titles and try to rephrase them in a way that you could understand what the article is about from looking at TOC of the article on right side

File: content/docs/competencies/modeling/architecture_frameworks/viewpoints_and_perspectives.md
Comment (ihardzeenka) [inline content/docs/competencies/modeling/architecture_frameworks/viewpoints_and_perspectives.md:1]: Think about adding more descriptive text to viewpoints and perspective sections. Maybe as simple as ad one more abstract description attribute.

File: content/docs/competencies/modeling/architecture_frameworks/c4_model.md
Comment (ihardzeenka) [inline content/docs/competencies/modeling/architecture_frameworks/c4_model.md:1]: 1. Make diagram types as header 3
2. Expand descriptions. Main concern is space between pictures. I would like it to be a bit bigger.
3. Static vs. Dynamic Diagrams. Brake into separate high-level sections. Add examples for dynamic.

**Comment by RGBD** [comment]
@ihardzeenka, requested changes uploaded

**Comment by ihardzeenka** [comment]
Closed in favor of https://github.com/software-architecture-guild/software-architecture-guild.github.io/pull/53

**Pull request closed without merging**